Assets/AnkrSDK/Plugins/WalletConnectSharp.Core/WalletConnectProtocol.cs

```diff
@@ -15,20 +15,7 @@ public abstract class WalletConnectProtocol : IDisposable
 	{
 		public static readonly string[] SigningMethods =
 		{
-			"eth_sendTransaction",
-			"eth_signTransaction",
-			"eth_sign",
-			"eth_signTypedData",
-			"eth_signTypedData_v1",
-			"eth_signTypedData_v2",
-			"eth_signTypedData_v3",
-			"eth_signTypedData_v4",
-			"personal_sign",
-			"wallet_silentSendTransaction",
-			"wallet_silentSignMessage",
-			"wallet_addEthereumChain",
-			"wallet_switchEthereumChain",
-			"wallet_updateEthereumChain"
+			"eth_sendTransaction", "eth_signTransaction", "eth_sign", "eth_signTypedData", "eth_signTypedData_v1", "eth_signTypedData_v2", "eth_signTypedData_v3", "eth_signTypedData_v4", "personal_sign", "wallet_silentSendTransaction", "wallet_silentSignMessage", "wallet_addEthereumChain", "wallet_switchEthereumChain", "wallet_updateEthereumChain"
```

Comment:
don't think this is a good change

Reply:
It's just a side effect of auto-formatting, I can roll it back